### PR TITLE
promethean number tweaks

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request

1. removes the inexplicable speed increase and item...not-slowdown, from prommies
2. nerfs their limb hp **(~~literally arbitrary placeholder values picked for rn til people come up with a comfy number, i think somewhere from 25 to 30 feels good and hits a good threshhold but who knows.~~) 25 is probably good**

## Why It's Good For The Game

we already spent a long time talking on discord about this but the tl;dr is that promethean is a really, really strong species at base, and has the added benefit of not suffering from baymed. 

event managers have expressed a bit of frustration at how strong the species is and i also agree w/ it. this is what was eventually settled on, while people work on adding other stuff to them in the background. (i think them having pain would also be cool since they're literally a fully biological species but that is neither here nor there, and it may be unironically rather crippling)

not really a 'fix' so much as a bandaid, as expressed by...someone  i forgor who

## Changelog

:cl:
balance: promethean number tweaks - removes random free speed buffs (almost half as fast as a taj bonus??) and decreases limb splodey threshhold
/:cl:

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
again, literally an arbitrary value picked for rn (reverted to what it previously was), waiting on response from folks on what might feel like a good spot

File: code/modules/organs/external/species/slime.dm [hunk 1]
```diff
@@ -13,57 +13,57 @@
 
 /obj/item/organ/external/arm/unbreakable/slime
 	nonsolid = 1
-	max_damage = 35
+	max_damage = 20
```
Comment (BlueWildrose) [inline code/modules/organs/external/species/slime.dm:16]:
```suggestion
	max_damage = 25
```

File: code/modules/organs/external/species/slime.dm [hunk 2]
```diff
@@ -13,57 +13,57 @@
...
 	spread_dam = 1
 	transparent = 1
 
 /obj/item/organ/external/arm/right/unbreakable/slime
 	nonsolid = 1
-	max_damage = 35
+	max_damage = 20
```
Comment (BlueWildrose) [inline code/modules/organs/external/species/slime.dm:22]:
```suggestion
	max_damage = 25
```

File: code/modules/organs/external/species/slime.dm [hunk 3]
```diff
@@ -13,57 +13,57 @@
...
 	spread_dam = 1
 	transparent = 1
 
 /obj/item/organ/external/leg/unbreakable/slime
 	nonsolid = 1
-	max_damage = 35
+	max_damage = 20
```
Comment (BlueWildrose) [inline code/modules/organs/external/species/slime.dm:28]:
```suggestion
	max_damage = 25
```

File: code/modules/organs/external/species/slime.dm [hunk 4]
```diff
@@ -13,57 +13,57 @@
...
 	spread_dam = 1
 	transparent = 1
 
 /obj/item/organ/external/leg/right/unbreakable/slime
 	nonsolid = 1
-	max_damage = 35
+	max_damage = 20
```
Comment (BlueWildrose) [inline code/modules/organs/external/species/slime.dm:34]:
```suggestion
	max_damage = 25
```

File: code/modules/organs/external/species/slime.dm [hunk 5]
```diff
@@ -13,57 +13,57 @@
...
 	spread_dam = 1
 	transparent = 1
 
 /obj/item/organ/external/foot/unbreakable/slime
 	nonsolid = 1
-	max_damage = 35
+	max_damage = 20
```
Comment (BlueWildrose) [inline code/modules/organs/external/species/slime.dm:40]:
```suggestion
	max_damage = 25
```

File: code/modules/organs/external/species/slime.dm [hunk 6]
```diff
@@ -13,57 +13,57 @@
...
 	spread_dam = 1
 	transparent = 1
 
 /obj/item/organ/external/foot/right/unbreakable/slime
 	nonsolid = 1
-	max_damage = 35
+	max_damage = 20
```
Comment (BlueWildrose) [inline code/modules/organs/external/species/slime.dm:46]:
```suggestion
	max_damage = 25
```

File: code/modules/organs/external/species/slime.dm [hunk 7]
```diff
@@ -13,57 +13,57 @@
...
 	spread_dam = 1
 	transparent = 1
 
 /obj/item/organ/external/hand/unbreakable/slime
 	nonsolid = 1
-	max_damage = 35
+	max_damage = 20
```
Comment (BlueWildrose) [inline code/modules/organs/external/species/slime.dm:52]:
```suggestion
	max_damage = 25
```

File: code/modules/organs/external/species/slime.dm [hunk 8]
```diff
@@ -13,57 +13,57 @@
...
 	spread_dam = 1
 	transparent = 1
 
 /obj/item/organ/external/hand/right/unbreakable/slime
 	nonsolid = 1
-	max_damage = 35
+	max_damage = 20
```
Comment (BlueWildrose) [inline code/modules/organs/external/species/slime.dm:58]:
```suggestion
	max_damage = 25
```

File: code/modules/organs/external/species/slime.dm [hunk 9]
```diff
@@ -13,57 +13,57 @@
...
 	spread_dam = 1
 	transparent = 1
 
 /obj/item/organ/external/head/unbreakable/slime	//They don't need this anymore.
 	nonsolid = 1
 	cannot_gib = 0
 	vital = 0
-	max_damage = 35
+	max_damage = 20
```
Comment (BlueWildrose) [inline code/modules/organs/external/species/slime.dm:66]:
```suggestion
	max_damage = 25
```

**Review by BlueWildrose — CHANGES_REQUESTED** [review]
As discussed in Discord, having it be 25 is probably best. It still is enough damage to cause instant limb break by a laser if a prommie has about 40% armor.

**Review by BlueWildrose — APPROVED** [review]

**Comment by timothyteakettle** [comment]
i'm pretty fine with these changes

**Pull request merged**